### PR TITLE
Fix keyboard test

### PR DESCRIPTION
**Pull request by aameijide**

### CODE REVIEW

File: VAS.Tests/Setup.cs [hunk 1]
```diff
@@ -58,6 +59,11 @@ public class ConfigDummy : Config
 		//Dummy class for VAS.Config
 	}
 
+	public class KeyboardDummy : Keyboard
```
Comment (vguzmanp) [inline VAS.Tests/Setup.cs]: Shouldn't this implement `IKeyboard` instead of inheriting `Keyboard`?

Comment (aameijide) [inline VAS.Tests/Setup.cs]: With that approach, we should implement our KeyboardDummy class, so it's better to derivate from Keyboard.

But, like Keyboard and KeyboardDummy are the same, it should be possible to instantiate Keyboard, but I prefer to follow the same pattern done with Config.

Comment (vguzmanp) [inline VAS.Tests/Setup.cs]: I don't really see the point of having empty classes that aren't actually Dummy, as they keep all the behaviour of their parents, but :+1:

**Comment by vguzmanp** [comment]
:+1:

**Pull request merged**